resources/entity_tree_form.ui

```diff
@@ -40,6 +40,13 @@
      <property name="bottomMargin">
       <number>1</number>
      </property>
+     <item>
+      <widget class="QComboBox" name="task_status_combo">
+       <property name="text">
+        <string>Task Status</string>
```

Comment:
_Task Status_ or _Task**s** Status_ ?

Reply:
I don't have a strong opinion. What should be the best?